nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -273,7 +273,7 @@ StaticLayer::incomingUpdate(map_msgs::msg::OccupancyGridUpdate::ConstSharedPtr u
 
 void
 StaticLayer::updateBounds(
-  double robot_x, double robot_y, double robot_yaw, double * min_x,
+  double /*robot_x*/, double /*robot_y*/, double /*robot_yaw*/, double * min_x,
```

Comment:
If these arguments are not used, how about removing them from the argument list? Can be outside the scope of the current PR, but something to consider.

Reply:
this function is from the virtual base class, so I'm not sure that is allowed